### PR TITLE
GH#23621 - Move PTP topic

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/23621

Preview: https://deploy-preview-28977--osdocs.netlify.app/openshift-enterprise/latest/networking/configuring-ptp.html

### CODE REVIEW

File: networking/configuring-ptp.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 == About PTP hardware
 
 {product-title} includes the capability to use Precision Time Protocol (PTP)hardware on your nodes.
-You can configure linuxptp services on nodes with PTP capable hardware.
+You can configure linuxptp services on nodes in your cluster that have PTP capable hardware.
```
Comment (bobfuru) [inline networking/configuring-ptp.adoc:15]: s/PTP capable/PTP-capable

File: networking/configuring-ptp.adoc [hunk 2]
```diff
@@ -21,11 +21,9 @@ The PTP Operator works with PTP capable devices on clusters provisioned only on
 
 You can use the {product-title} console to install PTP by deploying the PTP Operator. The PTP Operator creates and manages the linuxptp services. The Operator provides following features:
```
Comment (bobfuru) [inline networking/configuring-ptp.adoc:22]: s/provides following features/provides the following features

File: networking/configuring-ptp.adoc [hunk 3]
```diff
@@ -21,11 +21,9 @@ The PTP Operator works with PTP capable devices on clusters provisioned only on
...
 
-* Discover the PTP capable devices in cluster.
-* Manage configuration of linuxptp services.
-
-include::modules/nw-ptp-installing-operator.adoc[leveloffset=+1]
+* Discovery of the PTP capable devices in cluster.
```
Comment (bobfuru) [inline networking/configuring-ptp.adoc:24]: s/PTP capable/PTP-capable
s/in cluster/in a cluster/ (or "in-cluster") ?

**Review by bobfuru — COMMENTED** [review]
A few minor nits but otherwise LGTM.

**Comment by jboxman** [comment]
@bobfuru thanks!

**Pull request merged**